docs/guides/server/containers.adoc

```diff
@@ -147,7 +149,11 @@ To build the actual container image, run the following command from the director
 
 [source,bash]
 ----
-podman|docker build . -t mykeycloak
+podman build . -t mykeycloak
+# or for Docker (specify the Containerfile explicitly):
+docker build -f Containerfile . -t mykeycloak
+# or rename Containerfile to Dockerfile and use:
+docker build . -t mykeycloak
```

Comment:
@ahus1 IIRC, we also have a logic somewhere to replace `podman|docker` with just `podman`, correct? So this won't work for prod either, AFAICT.

Reply:
I'll check once I review that follow-up PR